docs/release-notes/8.0.35-27.md

```diff
@@ -0,0 +1,60 @@
+# Percona Server for MySQL 8.0.35-27 (2023-12-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.35-27](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.35 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html) in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+Percona Server for MySQL implements telemetry that fills in the gaps in our understanding of how you use Percona Server to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information. Find more information in the [Telemetry on Percona Server for MySQL](../telemetry.md) document.
+
+This merge fixes the following issue:
+
+* [PS-8849](https://jira.percona.com/browse/PS-8849): The server exited with the `Assertion failure: row0mysql.cc:218:len < 256 * 256 thread`.
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.35 and included in Percona Server for MySQL are the following:
+
+* Linked Open SSL library has been upgraded to OpenSSL 3.0.10
+* Removed the printed query string limit to display the characters for a detected deadlock section of the engine status log
+* The last detected deadlock section of the engine status log showed 1024 characters for the combined thread and query information. Removing the printed query string limit.
+* Fix for procession of single character tokens by a FTS parser plugin.
+
+### Deprecations and removals
+
+* The `binlog_transaction_dependency_tracking` server system variable is deprecated. The usage of this variable results in an warning. This variable will be removed in future releases.
+* The `old` and `new` server system variables are deprecated. The usage of these variables results in an warning. These variables will be removed in future releases.
+* The `--character-set-client-handshake` server variable is deprecated. The usage of this variable results in an warning. This variable will be removed in future releases.
+* `INFORMATION_SCHEMA.PROCESSLIST` is deprecated. The implementation of `SHOW PROCESSLIST` which uses that table is also deprecated. Use the Performance Schema implementation of `SHOW PROCESSLIST` instead. `performance_schema_show_processlist` is also deprecated.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.35 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html).
+
+## Bug fixes
+
+* [PS-7806](https://jira.percona.com/browse/PS-7806): Tables with column compression broke asynchronous replication.
+
+* [PS-8671](https://jira.percona.com/browse/PS-8671): The `innodb.ddl_kill` test failed on Jenkins.
```

Comment:
An intermittent failure of the `innodb.ddl_kill` test on Jenkins.

Reply:
fixed